src/riak_api_pb_server.erl

```diff
@@ -209,6 +209,32 @@ connected(timeout, State=#state{outbuffer=Buffer}) ->
     %% Flush any protocol messages that have been buffering
     {ok, Data, NewBuffer} = riak_api_pb_frame:flush(Buffer),
     {next_state, connected, flush(Data, State#state{outbuffer=NewBuffer})};
+connected({msg, 110, MsgData}, State) ->
```

Comment:
Macro is a good idea.

We can't handle it like other messages because those are all dispatched to other processes (PB listeners); we need *this* process to toggle the encoding flag.